src/front/app/components/Create/index.scss

```diff
@@ -0,0 +1,20 @@
+$create-blocks-title-color: #cccccc;
+$create-titles-text-color: #fafafa;
+$uppercase: uppercase;
+$disabled-title-checkbox-color: #cbcbcb;
+$enabled-title-checkbox-color: #000000;
+$checkbox-title-font: "Times New Roman";
+$create-header-title-color: #f6f6f6;
+$create-header-color: #666666;
+$create-header-button: #c5c5c5;
+$create-header-button-font: "Times New Roman";
+$create-header-button-size: 18px;
+$photo-download-color: #adadad;
+
```

Comment:
As we import this file to global index styles file - please consider using local variables per component to avoid possible conflicts (place them under the main wrapper class - I believe '.challenge-create' in your case)